internal/pkg/cli/deploy/static_site.go

```diff
@@ -205,18 +205,13 @@ func (d *staticSiteDeployer) convertSources() ([]manifest.FileUpload, error) {
 	return convertedFileUploads, nil
 }
 
-func (d *staticSiteDeployer) validateAlias() error {
-	if d.staticSiteMft.HTTP.Alias == "" {
+func (d *staticSiteDeployer) validateAppDomainAlias() error {
+	if d.staticSiteMft.HTTP.Alias == "" || d.staticSiteMft.HTTP.Certificate != "" {
 		return nil
 	}
 
-	hasImportedCerts := len(d.envConfig.HTTPConfig.Public.Certificates) != 0
-	if hasImportedCerts {
-		return fmt.Errorf("cannot specify alias when env %q imports one or more certificates", d.env.Name)
-	}
```

Comment:
Why are we removing this error? Is the intent just that the certificate for the env is ignored, and the static site requires a certificate to be imported in the `StaticSiteHTTP.Certificate` field?

Reply:
it's because this was wrong - static site creates its own dedicated cloudfront and don't use any imported cert at the env level.

Reply:
should we remove this error from the NLB validation as well then? https://github.com/aws/copilot-cli/blob/dc6d9535b9fb1cd5cc87975f158b980b85fdd88f/internal/pkg/cli/deploy/lbws.go#L273-L276